logger/logger.go

```diff
@@ -616,3 +571,73 @@ func getLabels(data map[string]interface{}) []attribute.KeyValue {
 
 	return attrs
 }
+
+//Log is a basic logging method that fulfills the WagClientLogger interface.
+func (l *Logger) Log(level wcl.LogLevel, title string, m map[string]interface{}) {
+	m["title"] = title
+	l.logWithLevel(LogLevel(level), m)
+}
+
+//NewConcreteLogger creates and returns a concrete logger instead of an interface
+//This was created because New() returned a KayveeLogger instead of a concrete type.
+func NewConcreteLogger(source string, contextValues M) *Logger {
```

Comment:
IMO, we should put a deprecation warning comment on `New` like
```go
// New creates a *logger.Logger. Default values are Debug LogLevel, kayvee Formatter, and std.err output.
//
// Deprecated: use NewConcreteLogger instead
func New(source string) ...
```

Also, this method should be called `NewConcreteLoggerWithContext`, and `NewConcreteLogger` should call be
```go
func NewConcreteLogger(source string) *Logger {
    return NewConcreteLoggerWithContext(source, nil)
}
```